src/utils/datetime.cljs

```diff
@@ -102,6 +103,14 @@
          (= (t/month now) (t/month datetime))
          (= (t/day now) (t/day datetime)))))
 
+(defn tomorrow?
+  [datetime]
+  (= (-> (t/now)
+         (t/plus
+          (t/days 1))
```

Comment:
This is slightly confusing. Due to the thread macro, it looks as if `(t/days 1)` is part of the pipeline, but it's just the argument to `t/plus`.

Reply:
Done